test/test-fs.c

```diff
@@ -1732,7 +1732,11 @@ TEST_IMPL(fs_realpath) {
   uv_fs_t req;
 
   loop = uv_default_loop();
+#ifdef __OpenBSD__
+  ASSERT(0 == uv_fs_realpath(loop, &req, "no_such_dir/no_such_file", dummy_cb));
```

Comment:
TBH I'm indifferent to this change but the last stance on this was trying to make them behave the same. See https://github.com/libuv/libuv/issues/1216

Reply:
I can just remove that commit for the time being and #1216 can be addressed separately.